common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -36,6 +36,10 @@
     private static final String PROP_LEVEL = "io.netty.leakDetectionLevel";
```

Comment:
Could we also support `io.netty.leakDetection.level`? We need to support the old name for backward compatibility, though.